hack/test-end-to-end.sh

```diff
@@ -358,6 +358,13 @@ echo "[INFO] Validating exec"
 registry_pod=$(osc get pod | grep deployment=docker-registry | grep docker-registry | awk '{print $1}')
 osc exec -p ${registry_pod} whoami | grep root
 
+# Diagnostics
+echo "[INFO] Running openshift-diagnostics"
+# show only errors, fail if any errors
+wait_for_command "openshift-diagnostics --config=${OPENSHIFTCONFIG} --master-config=${MASTER_CONFIG_DIR}/master-config.yaml --node-config=${NODE_CONFIG_DIR}/node-config.yaml --diaglevel=0" 60
```

Comment:
Shouldn't take 60 seconds. Just didn't want the test to randomly fail because you happen to have a lot of docker journal messages to analyze or something.

`openshift-diagnostics client` follows standard config loading. Plain `openshift-diagnostics` does not - it expects that config files are either specified or in a standard location. We can talk about this as I can see how it's confusing but divining user intent is tricky here...